app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -4,12 +4,12 @@ $(document).on("ready", function() {
   });
 
   $(".congress-message-rep-lookup").on("ajax:complete", function(xhr, data, status) {
-    $form = $(this);
+    var $form = $(this);
```

Comment:
`$form` is referenced in the other callback function (below), so it looks like this needs to be declared further up -- though I think it would also be fine if the `show_error` function found the form itself and didn't need it passed in as an argument.